bin/offline-dbg

```diff
@@ -133,6 +142,138 @@ EOF
     echo "**************************************************************"
 }
 
+get_highest_ofp_version() {
+    # if no protocol listed, default to OpenFlow14
+    hex_ofp_version=$(cat $cmd_dir/ovs-ofctl_-t_5_--version | grep "OpenFlow versions" | grep -oP [0-9]*x[0-9]* | tail -1)
+    version=${open_flow_protocols[$hex_ofp_version]}
+    if [ -z "$version" ]; then
+        echo "OpenFlow14"
+    else
+        echo $version
+    fi
+}
+
+do_collect-sos() {
+    OPTIND=1
+    while getopts "oh" opt; do
+        case ${opt} in
+            h)
+                usage
+                exit 0
+                ;;
+        esac
+    done
+    shift $(((OPTIND -1)))
+
+    if [ $# -lt 1 ]; then
+        usage
+        exit 1
+    fi
+    SOS=$1
+
+    cp $SOS ./sos.tar.xz || error "SOS archive file not found. Please specify a <sosreport>.tar.xz archive"
```

Comment:
don't use the current directory. Use $WOKDIR instead

Reply:
Also, if we are going to extract files from it, why not just extract from the original tarball and avoid the copy?